### PR TITLE
[ADS-355] information box component added

### PR DESCRIPTION
**Pull request by sharmajyo**
can be used for showing an information box.
it can accept a title, icon and children.

for issue-- Resolves https://github.com/Adslot/adslot-ui/issues/633

screen shot 2017-10-23 at 11 30 41 am

### CODE REVIEW

**Comment by sonhanguyen** [comment]
I think there is a `Banner` component in direct-web serving similar purpose. May as well checking that out in case we want to swap it out for this at some point.

File: src/components/adslot-ui/InformationBox/index.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    </div>
+    <div className="information-box-text">
+      {title ? <label className="information-box-title">{title}</label> : null}
+      <div className="information-box-node">{children}</div>
+    </div>
+  </div>;
+
+InformationBox.propTypes = {
+  children: PropTypes.node,
+  title: PropTypes.string,
```
Comment (sonhanguyen) [inline src/components/adslot-ui/InformationBox/index.jsx]: from the card's description this should be a `PropType.node`?

Comment (sharmajyo) [reply src/components/adslot-ui/InformationBox/index.jsx]: title isn't described in card.. I added this after a discussion with Gary.. I will update the issue

File: docs/examples/InformationBoxExample.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import Example from '../components/Example';
+import {
+  InformationBox,
+} from '../../src/dist-entry';
+
+class InformationBoxExample extends React.Component {
```
Comment (omgaz) [inline docs/examples/InformationBoxExample.jsx]: This can be a PureComponent

File: src/components/adslot-ui/InformationBox/styles.scss [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+.information-box-icon {
+  background-color: $color-white;
+  width: 70px;
+  height: 70px;
+  border-radius: $border-radius-circle;
+  text-align: center;
+  overflow: hidden;
+}
```
Comment (omgaz) [inline src/components/adslot-ui/InformationBox/styles.scss]: Can we nest the styles as in

```
.information-box {
  &-text {
    …
  }

  &-title {
    …
  }
}
```

File: src/components/adslot-ui/InformationBox/styles.scss [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+.information-box-title {
+  font-size: $font-size-subheader;
+  font-weight: $font-weight-bold;
+  color: $color-gray-darker;
+}
+
+.information-box-icon {
+  background-color: $color-white;
```
Comment (omgaz) [inline src/components/adslot-ui/InformationBox/styles.scss]: $color-background

File: src/components/adslot-ui/InformationBox/styles.scss [hunk 5]
```diff
@@ -0,0 +1,31 @@
+@import '~styles/color';
+@import '~styles/variable';
+
+.information-box {
+  background-color: $color-gray-lightest;
```
Comment (omgaz) [inline src/components/adslot-ui/InformationBox/styles.scss]: $color-background-highlighted

**Comment by sharmajyo** [comment]
@omgaz changes added

File: docs/examples/InformationBoxExample.jsx [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    return (<InformationBox
+      title="This is an information"
+      icon="/assets/svg-symbols.svg#cancel">
+        Content body.
+    </InformationBox>);
+  }
+}
+
+const exampleProps = {
+  componentName: 'InformationBox',
```
Comment (omgaz) [inline docs/examples/InformationBoxExample.jsx]: Information Box

File: src/components/adslot-ui/InformationBox/styles.scss [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+  &-title {
+    font-size: $font-size-subheader;
+    font-weight: $font-weight-bold;
+    color: $color-gray-darker;
+  }
+
+  &-icon {
+    background-color: $color-background;
+    width: 70px;
+    height: 70px;
```
Comment (omgaz) [inline src/components/adslot-ui/InformationBox/styles.scss]:
```
$information-box-icon-size = 70px;

background-color: $color-background;
width: $information-box-icon-size;
height: $information-box-icon-size;
```

File: docs/examples/InformationBoxExample.jsx (hunk 6 above)
Comment (sharmajyo) [reply docs/examples/InformationBoxExample.jsx]: most of the examples were following this pattern so I also went with majority :P I will change that

**Comment by omgaz** [comment]
Heya sorry, 2 more comments. I got distracted during review.

**Comment by sharmajyo** [comment]
changes added @omgaz

**Pull request merged**